blacklist/blacklist/Scenes/Debtors/DebtorsRouter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DebtorsRouter.swift
+//  blacklist
+//
+//  Created by Santiago Carmona Gonzalez on 2/22/18.
+//  Copyright © 2018 Opelty's Open Source Projects. All rights reserved.
+//
+
+import UIKit
+
+class DebtorsRouter: Router {
+    // Add transition methods here...
+
+    func goToContinueAction() { // TODO: Finish this
+        print("Going to continue screen...")
```

Comment:
Please remove this `print` we can use it to indicate something but when the main feature has implemented please remove it in order to avoid wasted info in the console.